src/lwc/geTemplateBuilder/geTemplateBuilder.js

```diff
@@ -92,6 +142,75 @@ export default class geTemplateBuilder extends NavigationMixin(LightningElement)
         }
     }
 
+    /*******************************************************************************
+    * @description Method builds and sorts a list of batch header fields for the
+    * child component geTemplateBuilderBatchHeader. Takes into consideration
+    * additionally required fields and exluded fields. List is used by the sidebar
+    * for the lightning-input checkboxes.
+    */
+    collectBatchHeaderFields() {
+        this.batchFields = mutable(this.diBatchInfo.fields);
+
+        Object.getOwnPropertyNames(this.batchFields).forEach((key) => {
+            let field = this.batchFields[key];
+
+            if (EXCLUDED_BATCH_HEADER_FIELDS.includes(field.apiName.replace(this.namespace, ''))) {
```

Comment:
What do you think about importing the fields so that we don't need to rely on manually replacing the namespace here?

Reply:
So we are actually importing the fields through `getObjectInfo` here. The reason I'm removing the namespace here is because we have the two hardcoded list of excluded and additionally required fields. Maybe a better way to handle this would be to handle namespacing on the hardcoded lists before they're used here?